include/exadg/incompressible_navier_stokes/spatial_discretization/operators/momentum_operator.cpp

```diff
@@ -42,7 +42,8 @@ MomentumOperator<dim, Number>::initialize(MatrixFree<dim, Number> const &   matr
     this->viscous_kernel = std::make_shared<Operators::ViscousKernel<dim, Number>>();
     this->viscous_kernel->reinit(matrix_free,
                                  operator_data.viscous_kernel_data,
-                                 operator_data.dof_index);
+                                 operator_data.dof_index,
+                                 operator_data.quad_index);
```

Comment:
Same as above: Add parameter `MomentumOperatorData::quad_index_standard` initialized in `IncNS::SpatialOperatorBase` as `data.quad_index_standard = get_quad_index_velocity_linear();` and use this parameter in the line above, i.e. `operator_data.quad_index_standard`.